src/components/organisms/TimingEvent/TimingEvent.tsx

```diff
@@ -1,80 +1,114 @@
-  setShowCreateEventModal: () => void;
-  setEditedEvent: (event: WorldEvent) => void;
+  space?: SpaceWithId;
 };
 
-export const TimingEvent: React.FC<TimingEventProps> = ({
-  event,
-  setShowCreateEventModal,
-  setEditedEvent,
-}) => {
-  const [display, setDisplay] = useState(false);
-
-  const { findVenueInRelatedVenues } = useRelatedVenues();
-  const space = findVenueInRelatedVenues({ spaceId: event.spaceId });
+export const TimingEvent: React.FC<TimingEventProps> = ({ event, space }) => {
+  const {
+    isShown: showDeleteEventModal,
+    show: setShowDeleteEventModal,
+    hide: setHideDeleteEventModal,
+  } = useShowHide();
 
-  const showButton = (e: React.MouseEvent) => {
-    e.preventDefault();
-    setDisplay(true);
-  };
-  const hideButton = (e: React.MouseEvent) => {
-    e.preventDefault();
-    setDisplay(false);
-  };
+  const {
+    isShown: isShownCreateEventModal,
+    show: showCreateEventModal,
+    hide: hideCreateEventModal,
+  } = useShowHide();
 
   return (
-    <div
-      key={event.id}
-      className="TimingEvent"
-      onMouseEnter={showButton}
-      onMouseLeave={hideButton}
-    >
-      <div className="TimingEvent__time">
-        <p>{format(eventStartTime({ event }), "do MMM")}</p>
-        <p className="TimingEvent__time-start">
-          {formatTimeLocalised(eventStartTime({ event }))}
-        </p>
-        <p>{formatTimeLocalised(eventEndTime({ event }))}</p>
-      </div>
-      <div className="TimingEvent__details">
-        <p>
-          <span className="TimingEvent__details-name">{event.name}</span> by{" "}
-          <span className="TimingEvent__details-host">{event.host}</span>
-        </p>
-        <p className="TimingEvent__details-description">{event.description}</p>
-        <p className="TimingEvent__details-room">
-          in <span className="event-details-room-name">{space?.name}</span>
-        </p>
-      </div>
+    <>
+      <div key={event.id} className="">
+        <div className="bg-white divide-y divide-gray-200">
+          <div className="flex justify-start items-start">
+            <div className="px-6 py-4 w-96">
+              <div className="text-sm font-medium text-gray-900">
+                {format(eventStartTime({ event }), "do MMM")}
+              </div>
+              <div className="text-sm text-gray-500">
+                {formatTimeLocalised(eventStartTime({ event }))} -{" "}
+                {formatTimeLocalised(eventEndTime({ event }))}
+              </div>
+            </div>
+            <div className="px-6 py-4 grow w-full flex flex-col flex-auto">
+              <div className="text-sm text-gray-900">
+                <span className="font-medium">{event.name}</span> by{" "}
+                <span className="font-medium">{event.host}</span>
+              </div>
+              {/* <p className="TimingEvent__details-description">{event.description}</p> */}
+              <p className="TimingEvent__details-room">
+                in{" "}
```

Comment:
Another `{" "}` I can't determine by just looking at the code if intentional or not